ui/src/templates/api/index.ts

```diff
@@ -492,3 +494,13 @@ export const installTemplate = async (orgID: string, templateUrl: string) => {
 
   return applyTemplates(params)
 }
+
+export const fetchStacks = async (orgID: string) => {
+  const resp = await getStacks({query: {orgID}})
+
+  if (resp.status >= 300) {
+    throw new Error((resp.data as PkgError).message)
+  }
+
+  return (resp.data as {stacks: InstalledStack[]}).stacks
```

Comment:
but they're not `InstalledStack` right?, the BE returns `Stack` and then you have to apply properties we want on FE?

Reply:
@ebb-tide correct! But I'm working with Johnny on this to get the API cleaned up around stuff like this.

Reply:
And here, I just wanted to keep API honest- if it works it's cool with me